src/call.c

```diff
@@ -790,8 +793,18 @@ int call_alloc(struct call **callp, const struct config *cfg, struct list *lst,
 	call->af     = prm->af;
 	call->ansadir = SDP_SENDRECV;
 	call->ansvdir = SDP_SENDRECV;
+	call->support_replaces = false;
 	call_decode_sip_autoanswer(call, msg);
 
+	/* check if replaces is supported on incoming calls */
+	hdr = sip_msg_hdr(msg, SIP_HDR_SUPPORTED);
+	if (hdr) {
+		if (!re_regex(hdr->val.p, hdr->val.l, "replaces", &v)) {
```

Comment:
```c
if (hdr && !re_regex(hdr->val.p, hdr->val.l, "replaces", &v)) {
```

Reply:
I changed it in favor of `sip_msg_hdr_has_value`